tests/test_rest/test_api_gateway_event.py

```diff
@@ -0,0 +1,95 @@
+import uuid
+from unittest.mock import ANY, MagicMock, patch
+
+import pytest
+
+from lbz.rest import APIGatewayEvent
+
+
+class TestAPIGatewayEvent:
+    @patch.object(uuid, "uuid4")
+    @pytest.mark.parametrize("uuid4", ["uuid-1", "uuid-2", "uuid-3"])
+    def test_builds_basic_version_of_simulated_event(
+        self, mocked_uuid4: MagicMock, uuid4: str
+    ) -> None:
+        mocked_uuid4.return_value = uuid4
```

Comment:
Parameterizing uuid is pointless, especially if values are almost the same:
```suggestion
    def test_builds_basic_version_of_simulated_event(self, mocked_uuid4: MagicMock) -> None:
        mocked_uuid4.return_value = "<mocked-uuid-value>"

```